recipes/nb_conda/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+fontconfig
```

Comment:
No worries. I believe it was `phantomjs` in the other cases. Though I don't see it in the requirements list.

Do we actually need this, @bollwyvl?